src/Sulu/Bundle/ContentBundle/Tests/Functional/Controller/NodeControllerTest.php

```diff
@@ -828,51 +968,6 @@ public function testPutRemoveShadowWithDifferentTemplate()
         $this->assertEquals('overview', $response['template']);
     }
 
-    public function testPutState()
-    {
-        $data = [
-            'title' => 'Testtitle',
-            'template' => 'default',
-            'url' => '/test',
-        ];
-
-        $client = $this->createAuthenticatedClient();
-
-        $client->request('POST', '/api/nodes?webspace=sulu_io&language=en', $data);
-        $response = json_decode($client->getResponse()->getContent(), true);
-        $this->assertEquals(1, $response['nodeState']);
-
-        $client->request('PUT', '/api/nodes/' . $response['id'] . '?webspace=sulu_io&language=en&state=2', $data);
-        $client->request('GET', '/api/nodes/' . $response['id'] . '?language=en', $data);
-        $response = json_decode($client->getResponse()->getContent(), true);
-        $this->assertEquals(2, $response['nodeState']);
-
-        $client->request('PUT', '/api/nodes/' . $response['id'] . '?webspace=sulu_io&language=en&state=1', $data);
-        $client->request('GET', '/api/nodes/' . $response['id'] . '?language=en', $data);
-        $response = json_decode($client->getResponse()->getContent(), true);
-        $this->assertEquals(1, $response['nodeState']);
-    }
-
-    public function testPutWithoutState()
-    {
-        $data = [
-            'title' => 'Testtitle',
-            'template' => 'default',
-            'url' => '/test',
-        ];
-
-        $client = $this->createAuthenticatedClient();
-
-        $client->request('POST', '/api/nodes?webspace=sulu_io&language=en&state=2', $data);
-        $response = json_decode($client->getResponse()->getContent(), true);
-        $this->assertEquals(2, $response['nodeState']);
-
-        $client->request('PUT', '/api/nodes/' . $response['id'] . '?webspace=sulu_io&language=en', $data);
-        $client->request('GET', '/api/nodes/' . $response['id'] . '?language=en', $data);
-        $response = json_decode($client->getResponse()->getContent(), true);
-        $this->assertEquals(2, $response['nodeState']);
-    }
```

Comment:
Not needed anymore because of https://github.com/sulu/sulu/pull/2361#discussion_r69082311